### PR TITLE
Pull analytics from linked fields in webflow

### PR DESCRIPTION
**Pull request by michael616kriel**
https://offerzen.atlassian.net/jira/software/projects/GPS/boards/4?assignee=61309abdac61dc00696a26e4&selectedIssue=GPS-93
Screenshot 2022-04-22 at 15 11 30
7

### CODE REVIEW

File: offerzen/global/employers/employers@4.0.js [hunk 1]
```diff
@@ -37,12 +37,18 @@ window.$loaded(function (window, document, $, undefined) {
...
+      var category = $(".js-analytics-category").text();
+      var source = $(".js-analytics-source").text();
+      
       dataLayer.push({
-        event: 'Company Lead Form Submitted ',
-        action: 'Lead Form Submitted',
-        label: 'Company Sign Up / Employer Landing Page',
-        category: 'Core',
-        source: 'Demand Sign Up',
+        event: event || 'Company Lead Form Submitted ',
```
Comment (MarkusKuhn) [inline offerzen/global/employers/employers@4.0.js:47]: @michael616kriel nice one on the defaults here! 👌

**Review by MarkusKuhn — APPROVED** [review]
Nice one @michael616kriel !

File: offerzen/global/employers/employers@4.0.js (hunk 1 above)
Comment (jethroflanagan) [inline offerzen/global/employers/employers@4.0.js:47]:
```suggestion
        event: event || 'Company Lead Form Submitted',
```

**Review by jethroflanagan — CHANGES_REQUESTED** [review]
Looks good, please just drop the space on one of the defaults.

**Review by jethroflanagan — APPROVED** [review]

**Pull request merged**